### ISSUE TITLE
Changes made by the rules engine don't show up in logs

### ISSUE THREAD

**Issue by hobinjk** [mozilla-iot/gateway#1829]
1. Create rule to turn light on at 7 pm and turn it off at 11 pm
2. Create log of light to admire this rule's function
3. Log only shows human-created changes

**Comment by hobinjk** [comment]
Weirdly I can't reproduce this with a simple "While switch is on turn light on" rule

**Comment by hobinjk** [comment]
Looking at this further it appears that the actual issue I saw here was that the light's state was desynced due to a limitation of the philips hue adapter where it doesn't get updates from lights that are disconnected (usually this means turned off at the switch). This meant that my light was considered "on" but the transition from on to off using a time rule actually was logged properly

**Comment by madb1lly** [comment]
Hi @hobinjk, this seems to be how my logs are being recorded too. 

For example, I know that my device was set at 3 last Thursday, then the rules-engine set it to 2 in evening and to 3 the next morning, but neither of these changes appear in the logs. However, I manually changed it to 0 later on Friday and back to 3 again today and these changes appear on the logs graph.

I think this partly explains the strange behaviour I was seeing which I reported in IRC and on Discourse last week.

Cheers :)
